src/rules/function-blacklist/__tests__/index.js

```diff
@@ -0,0 +1,47 @@
+import {
+  ruleTester,
+  warningFreeBasics,
+} from "../../../testUtils"
+import rule, { ruleName, messages } from ".."
+
+const testRule = ruleTester(rule, ruleName)
+
+testRule([
+  "rgba",
+  "scale",
+  "linear-gradient",
+], tr => {
+  warningFreeBasics(tr)
+  tr.ok("a { color: pink; }")
+
```

Comment:
Aww yes, that would be helpful. I got a bit distracted with the the whole index thing...